incubator/hnc/api/v1alpha2/hierarchy_types.go

```diff
@@ -35,7 +35,8 @@ const (
 	LabelInheritedFrom        = MetaGroup + "/inheritedFrom"
 	FinalizerHasSubnamespace  = MetaGroup + "/hasSubnamespace"
 	LabelTreeDepthSuffix      = ".tree." + MetaGroup + "/depth"
-	AnnotationManagedBy       = MetaGroup + "/managedBy"
+	AnnotationManagedBy       = MetaGroup + "/managed-by"
+	AnnotationManagedByOld    = MetaGroup + "/managedBy"
```

Comment:
nit: When we changed to subnamespace-of we used deprecated instead of old. 

https://github.com/kubernetes-sigs/multi-tenancy/blob/b6089fff3909031ae593a739135474c4f9c173aa/incubator/hnc/api/v1alpha2/subnamespace_anchor.go#L28

Reply:
Also, should we add a comment/issue link indicating when/if we can remove this old/deprecated annotation?

Reply:
Changed both to the more descriptive "V1A1" suffix and added link to issue.